examples/mock_recipient/mock_test.go

```diff
@@ -26,72 +23,64 @@ var ganachePrivateKeys = []string{
 	"646f1ce2fdad0e6deeeb5c7e8e5543bdde65e86029e2fd9fc169899c440a7913",
 }
 
-func setupAuth(t *testing.T) (*bind.TransactOpts, *ethclient.Client, *ecdsa.PrivateKey) {
-	ec, err := ethclient.Dial(common.DefaultEthEndpoint)
-	require.NoError(t, err)
-	t.Cleanup(func() {
-		ec.Close()
-	})
-	chainID, err := ec.ChainID(context.Background())
-	require.NoError(t, err)
-
-	pk, err := ethcrypto.HexToECDSA(ganachePrivateKeys[0])
-	require.NoError(t, err)
-
-	auth, err := bind.NewKeyedTransactorWithChainID(pk, chainID)
-	require.NoError(t, err)
-	return auth, ec, pk
-}
-
 func TestMock_Execute(t *testing.T) {
 	ctx, cancel := context.WithTimeout(context.Background(), time.Second*30)
 	defer cancel()
 
-	auth, conn, pk := setupAuth(t)
-	chainID, err := conn.ChainID(ctx)
+	pk, err := ethcrypto.HexToECDSA(ganachePrivateKeys[0])
```

Comment:
This test isn't using the gsnforwarder, and I ended up spending a lot of time to debug it, that could have been used for atomic swap issues. It was reusing the same txOpts on multiple transactions and it wasn't setting the nonce in the transfer.